### PR TITLE
add easyconfigs for IMB with different toolchains

### PR DESCRIPTION
**Pull request by boegel**
@wpoely86: please review?

### CODE REVIEW

File: easybuild/easyconfigs/i/IMB/IMB-4.0.1-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ global communication operations for a range of message sizes"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://software.intel.com/sites/default/files/managed/34/aa/']
+sources = ['%(name)s_%(version)s.tgz']
+
+prebuildopts = "cd src && "
+buildopts = "all"
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/IMB/IMB-4.0.1-intel-2015a.eb]: no need to pass `MPI_HOME=$EBROOTOPENMPI` here?

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/fec5f4e3eb81ed0809e7 for a full test report.

File: easybuild/easyconfigs/i/IMB/IMB-4.0.1-intel-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/i/IMB/IMB-4.0.1-intel-2015a.eb]: no, building IMB on top of Intel MPI works, it uses the `make_ict` (intel cluster tools) Makefile by default, which uses `mpiicc` (as opposed to `mpicc`)

**Comment by boegel** [comment]
going in, thanks @wpoely86!

**Pull request merged**